src/vector/core/lorentz/xyzt.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019-2020, Jim Pivarski, Eduardo Rodrigues, and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/vector for details.
+
+
+# Functions that return a value
+
+def pt(rec):
+    return np.sqrt(rec.x**2 + rec.y**2)
+
+def eta(rec):
```

Comment:
One can be even more concise here and use `pt` in the denominator.

BTW, do we want to test for `pt==0` and return `inf`? Seems appropriate unless numpy does that for you automatically when calculating the ratii array.

Reply:
It does:

```python
In [2]: np.array([1]) / np.array([0])
/usr/local/Cellar/ipython/7.12.0/libexec/bin/ipython:1: RuntimeWarning: divide by zero encountered in true_divide
  #!/usr/local/opt/python/bin/python3.7
Out[2]: array([inf])
```

Reply:
OK, great - had not checked.